src/service/resources-impl.js

```diff
@@ -1070,6 +1036,9 @@ export class Resources {
         task.promise.then(this.taskComplete_.bind(this, task, true),
             this.taskComplete_.bind(this, task, false))
             .catch(reportError);
+        if (task.resource.getState() == ResourceState.LAYOUT_SCHEDULED) {
```

Comment:
It was done in the `Resource` class by calling internal `Resources` state. So, I decided to move it here - it's the same signal, just less leaking of `Resources` vars.